prov/verbs/src/verbs_info.c

```diff
@@ -849,21 +849,15 @@ static int fi_ibv_copy_ifaddr(const char *name, const char *service, uint64_t fl
 	}
 	ret = rdma_create_ep(&id, rai, NULL, NULL);
 	if (!ret) {
-		for (fi = info; fi; fi = fi->next)
+		for (fi = info; fi; fi = fi->next) {
 			if (!strncmp(id->verbs->device->name, fi->domain_attr->name,
-						strlen(id->verbs->device->name)))
-				break;
-		if (!fi) {
-			FI_WARN(&fi_ibv_prov, FI_LOG_FABRIC,
-					"No matching fi_info for device: "
-					"%s with address: %s\n",
-					id->verbs->device->name, name);
```

Comment:
@hoopoepg , I'm not sure if I understand you. The warning is raised when we have a valid verbs interface corresponding to id->verbs but no matching fi_info based on the domain attribute name. Are you seeing this warning for an interface that's not a valid verbs device?